### PR TITLE
[FIXED] Internal connections were counted toward account's max connections (client or leaf)

### PR DESCRIPTION
**Pull request by matthiashanel**
Question, what's the expected influence of JS on conn count?

### CODE REVIEW

File: server/events.go [hunk 1]
```diff
@@ -1613,7 +1613,7 @@ func (s *Server) accConnsUpdate(a *Account) {
 	if !s.eventsEnabled() || a == nil {
 		return
 	}
-	s.sendAccConnsUpdate(a, fmt.Sprintf(accConnsEventSubjOld, a.Name), fmt.Sprintf(accConnsEventSubjNew, a.Name))
+	s.sendAccConnsUpdate(a, fmt.Sprintf(accConnsEventSubjNew, a.Name))
```
Comment (kozlovic) [inline server/events.go:1616]: Why remove "old" subject? Would that possibly break apps listening to old subject? Don't mind dropping, but maybe not in 2.6.1...

**Review by kozlovic — CHANGES_REQUESTED** [review]
Don't want to drop "old" subject in this update release, I think..

**Comment by matthiashanel** [comment]
It would

File: server/events.go (hunk 1 above)
Comment (matthiashanel) [reply server/events.go:1616]: It would

**Review by matthiashanel — COMMENTED** [review]
Fine with changes. But would not remove the backwards compatibility subject

File: server/events.go (hunk 1 above)
Comment (derekcollison) [reply server/events.go:1615]: Possibly if apps were listening to those updates, but was messing up the accounting logic even beyond the counting JETSTREAM and ACCOUNT clients internally.

Comment (derekcollison) [reply server/events.go:1615]: We could send to both if you both feel strongly, but we can't process both as subs..

Comment (derekcollison) [reply server/events.go:1615]: I personally have no problem removing all together, but will trust your judgement here. But we can't process both for sure.

Comment (matthiashanel) [reply server/events.go:1615]: we can still send them all we have to do is not subscribe in events.go line 780.

given this comment:
```
// accConnsUpdate is called whenever there is a change to the account's
// number of active connections, or during a heartbeat.
func (s *Server) accConnsUpdate(a *Account) {
```
how is processing the message twice different from a change in account followed by a heartbeat?

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**